Documentation/ExtensionConfiguration/Index.rst

```diff
@@ -55,6 +28,26 @@ followed by a forward slash and the lowercase extension name with minus scores.
    If composer does not find your site-package extension check if you are
    using the correct separation chars in the correct places.
 
+.. _extension-manager-configuration:
+
+Extension declaration file :file:`ext_emconf.php`
+=================================================
+
+Since version 11 this file is only mandatory if you are not using Composer.
```

Comment:
Maybe we should add the hint that if you want to publish the extension to the TER, the emconf is still needed (as far as I know).

Reply:
Since when would you publish a site package extension to the TER?

Reply:
You most likely wouldn't (though I'm not sure if the themes flowing around aren't technically sitepackages) - it was meant as a more general hint, that there is another case where it is mandatory because the paragraph sounded as if it's about the emconf in general.

Reply:
I think this is a tutorial for developers to create their own site package for local use here. We also don't tell them how to get the site package into packagist. The TER is just not in the scope and would probably confuse more then help. There are plenty of places talking about extensions and how to publish them.

Reply:
ok.